SPM_Node.X/SPM.c

```diff
@@ -1,10 +1,167 @@
 #include "SPM.h"
 
+float analog_channels[36] = {0};
+
 void main(void){
 	init_general();// Set general runtime configuration bits
 	init_gpio_pins();// Set all I/O pins to low outputs
 	init_oscillator();// Initialize oscillator configuration bits
 	init_timer2();// Initialize timer2 (millis)
-    
-    while(1){}
-}
\ No newline at end of file
+
+	while(1){
+		update_analog_channels();
+
+    if(millis-canAnalogMillis > 25){
+      CANAnalogChannels();
+    }
+	}
+}
+
+void CANAnalogChannels(void){
+	CAN_data data = {0};
+  int i;
+  for(i = 0;i<9;i++){
+	  data.halfword0 = (uint16_t) (analog_channels[i*4]*ANALOG_CAN_SCL);
+	  data.halfword1 = (uint16_t) (analog_channels[(i*4)+1]*ANALOG_CAN_SCL);
+	  data.halfword2 = (uint16_t) (analog_channels[(i*4)+2]*ANALOG_CAN_SCL);
+	  data.halfword3 = (uint16_t) (analog_channels[(i*4)+3]*ANALOG_CAN_SCL);
+	  CAN_send_message(ANALOG_REAR_ID + i, 8, data);
```

Comment:
Yeah I was just doing this as a temporary thing, not sure how you wanted me to change the can spec